app/src/main/java/com/odysee/app/model/Claim.java

```diff
@@ -18,6 +18,7 @@
 import java.text.NumberFormat;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
+import java.time.ZonedDateTime;
```

Comment:
ZonedDateTime was added in Android API Level 26 and Odysee Android must run on devices running API Level 21 or higher

Reply:
Does this fix it: https://stackoverflow.com/a/60687913/15603854 (this bit of code is currently in the project)

I didn't get compile errors like I normally do with other API Level problems, so I think it's related to this.

Reply:
Oh. Sorry. You are right.